fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -0,0 +1,4 @@
+PixelDroid is a Free/Libre and Open Source Android client for Pixelfed, the
```

Comment:
At least at F-Droid, line breaks will be replaced by `<br>`s – so better keep a paragraph "single line".